octoprint_discordremote/discord.py

```diff
@@ -285,8 +285,10 @@ def handle_dispatch(self, js):
             for upload in data['attachments']:
                 filename = upload['filename']
                 url = upload['url']
-                snapshots, embeds = self.command.download_file(filename, url, user)
-                self.send(embeds=embeds)
+
+                if filename.endswith((".g", ".gco", ".gcode")):
```

Comment:
This will most likely stop the bot from downloading any zip files, I suggest:
    if filename.endswith((".g", ".gco", ".cgode", ".zip")) or filename[-4:].endswith(".zip"):

Reply:
`filename.endswith(".zip")` is same with `filename[-4:].endswith(".zip")`.

I'll just add ".zip" to extensions tuple.

Reply:
filename.endswith(".zip") 

will throw false for multivolume zip files (*.zip.001, *.zip.002, ...), 

    filename[-4:].endswith(".zip") 

will let those files through.

Reply:
Oh, I got it.

Reply:
Thanks, looking good :)